packages/xod-arduino/platform/program.tpl.cpp

```diff
@@ -115,8 +132,13 @@ void handleTweaks() {
 
           {{/eachLinkedTweakNode}}
         }
-
-        XOD_DEBUG_SERIAL.find('\n');
+        {{#forLinkedTetheringInetNode nodes}}
+        if (!tetheringInetNodeReceivingData) {
+          XOD_DEBUG_SERIAL.find('\n');
+        }
+        {{else}}
+          XOD_DEBUG_SERIAL.find('\n');
```

Comment:
```suggestion
        XOD_DEBUG_SERIAL.find('\n');
```

Too much indentation. See the generated fixtures